show/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# © 2019 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import show_place
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['.show_place' imported but unused](https://app.codacy.com/app/numigi/odoo-recording-addons/pullRequest?prid=4660575)

Reply:
@ddufresne  all `imported but unused`should be ignored , right ?

Reply:
Yes, these messages are generated automatically by Codacy. Some of them are irrelevant in Odoo.